### DISCUSSION TITLE
64-character limitation on the IP field in Suricata 6

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#4693] [Unsupported Versions]
Hello!

According to the SO Documentation, Suricata has currently a  64-character limitation on the IP field if you try to work with [thresholds](https://docs.securityonion.net/en/2.3/managing-alerts.html#threshold). However, if I try to use the described workaround and split my IP field like this

thresholding:
  sids:
    2012454:
      - suppress:
          gen_id: 1
          track: by_dst
          ip: 1.1.1.1,2.2.2.2,3.3.3.3,4.4.4.4,5.5.5.5,6.6.6.6,7.7.7.7,8.8.8.8
      - suppress:
          gen_id: 1
          track: by_dst
          ip: 9.9.9.9,10.10.10.10,11.11.11.11

my Suricata can't render the config and throws the following error:

2021-07-05 07:07:57,703 [salt.pillar:896 ][CRITICAL][1936] Rendering SLS 'minions.my-sensor_sensor' failed, render error:
mapping values are not allowed in this context

Does anybody have a working configuration for suppressing an alarm for multiple IP? Thanks in advance!

Cheers, Ben

**Comment by onezeroecho** [comment]
I'm suppressing alerts with multiple IPs. Try using single quotes to enclose the comma-delimited list:
```yaml
ip: '9.9.9.9,10.10.10.10,11.11.11.11'
```

**Comment by secuser0965453** [comment]
Hello,
Did you make sure to keep formatting ?
Same leading spaces ?
I do not need any quotes in my rules.

**Reply by ben-sec** [reply]
Do you exceed 64 chars for one of your IP fields?

**Comment by ben-sec** [comment]
Well, my suppress rules with multiple IP do not work (they render correctly and I'm very carefull regarding formatting), with or without quotes. If the 64-character limitation for the IP field is still an issue, than I'm wondering if there is a working workaround?
